src/test/java/org/mskcc/cbio/portal/integrationTest/scripts/TestImportCnaDiscreteLongData.java

```diff
@@ -326,7 +319,6 @@ public void testImportCnaDiscreteLongDataAddsGeneticAlterationsFromNonCnaEvents(
     @Test
     public void testImportCnaDiscreteLongDataIgnoresLineWithDuplicateGene() throws Exception {
         List<TestGeneticAlteration> beforeGeneticAlterations = getAllGeneticAlterations();
-        assertEquals(57, beforeGeneticAlterations.size());
```

Comment:
should the `beforeGeneticAlterations` variable also be removed? I think so...it is not used anymore. Or you could use it at the end to verify that `afterGeneticAlterations == beforeGeneticAlterations+1` ?

Reply:
Done https://github.com/cBioPortal/cbioportal-core/pull/46/commits/90aba7e9f33e7e6ea7d3f59bb7e6356329192599